cpu/sam0_common/periph/cpuid.c

```diff
@@ -39,6 +39,8 @@
 
 void cpuid_get(void *id)
 {
+    /* Use memcpy to prevent unaligned access, which is not supported on
+     * cortex-m0+, to id */
```

Comment:
AFAIK unaligned access is not supported on any ARM core btw.

Reply:
I remember this being a limitation only on the ARMv6-M and not on the ARMv7-M. I'll try to verify this